.github/workflows/win-package-test.yml

```diff
@@ -321,88 +223,18 @@ jobs:
           name: msi-build
           path: ./dist
 
-      - name: Ensure required ports in the dynamic range are available
-        run: |
-          $ErrorActionPreference = 'Continue'
-          & ${{ github.workspace }}\.github\workflows\scripts\win-required-ports.ps1
-
-      - name: Install the collector
+      - name: Set the MSI_COLLECTOR_PATH environment variable
         run: |
           $ErrorActionPreference = 'Stop'
           $msi_path = Resolve-Path .\dist\splunk-otel-collector*.msi
           Test-Path $msi_path
-          Write-Host "Installing $msi_path ..."
-          $process = Start-Process -Wait -PassThru msiexec "/i `"$msi_path`" /qn /l*v msi-install.log SPLUNK_ACCESS_TOKEN=1stInstall SPLUNK_REALM=1st"
-          if ($process.ExitCode -ne 0) {
-            throw "MSI installation failed with exit code $($process.ExitCode)"
-          }
-
-      - name: "In case of failure: display the install logs"
-        if: ${{ failure() }}
-        run: Get-Content msi-install.log
+          "MSI_COLLECTOR_PATH=$msi_path" | Out-File -FilePath $env:GITHUB_ENV -Append
 
-      - name: Start the collector service
+      - name: Run the MSI tests
         run: |
-          $sc = Start-Service splunk-otel-collector -PassThru
-          $sc.WaitForStatus('Running', '00:00:30')
+          go test -timeout 15m -v github.com/signalfx/splunk-otel-collector/tests/msi
```

Comment:
Should we use the `setup-go` action, e.g.
https://github.com/signalfx/splunk-otel-collector/blob/main/.github/workflows/windows-test.yml#L43-L47, instead of the default installed on the runner?

Reply:
The test doesn't relate to the go version used to build the collector. I opted to use the one that comes on board with the runner to save any possible download and install time. It is technically possible that we can use a feature on the test code not present on the default runner version, but, unless that actually happens any download and install cost seems unjustified to me.

BTW, thanks for looking even after merge, I appreciate that!